src/vs/workbench/contrib/notebook/browser/diff/diffComponents.ts

```diff
@@ -58,6 +60,37 @@ export function getOptimizedNestedCodeEditorWidgetOptions(): ICodeEditorWidgetOp
 	};
 }
 
+export class CellDiffPlaceholderElement extends Disposable {
```

Comment:
Base view model has nothing to dow with cells.
Using this we can now have a view model for diffing of notebook level metadata as well